superset/viz.py

```diff
@@ -372,9 +372,7 @@ def query_obj(self) -> QueryObjectDict:  # pylint: disable=too-many-locals
             del groupby[groupby_labels.index(DTTM_ALIAS)]
             is_timeseries = True
 
-        granularity = self.form_data.get("granularity") or self.form_data.get(
```

Comment:
This is somewhat concerning as these terms mean different things in the form-data.

Reply:
Agreed, good reminder to NEVER have dual meanings for variables.